[DOCS] CLI Edits (#7865)
great-expectations/great_expectations: docs/docusaurus/docs/guides/setup/configuring_metadata_stores/components_how_to_configure_an_expectation_store_in_amazon_s3/_confirm_list.mdx

> If you followed the optional step to copy your existing Expectation Suites to the S3 bucket, you can confirm that Great Expectations can find them by running the following Python code:

Reviewer: ```suggestion
If you copied your existing Expectation Suites to the S3 bucket, run the following Python code to confirm that Great Expectations can find them:
```

```python
import great_expectations as gx

context = gx.get_context()
context.list_expectation_suite_names()
```

> Your output should include the Expectations you copied to Amazon S3. If you did not copy Expectations to the new Store, they will not be listed.

Reviewer: ```suggestion
Your output should include the Expectations you copied to S3. Expectations that weren't copied to the new Store aren't listed.
```
